crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -103,7 +103,11 @@ def dump_openstack_database
       node.save
     end
 
-    raise "There does not seem to exist any node running the database." if db_nodes.empty?
+    # Check if database proposal exists in case when somebody is
+    # trying to upgrade undeployed nodes or only with Ceph cluster
+    db_proposal = Proposal.where(barclamp: "database").first
```

Comment:
MINOR: you could also use 
`db_proposal = Proposal.find_by(barclamp: "database")` 
or
`db_proposal = Proposal.find_by(barclamp: "database", name: "default")`